utils/src/check/clobber-functions.c

```diff
@@ -127,6 +127,19 @@ static struct clobber_function clobber_pb_sb_bad_flag = {
 	&do_clobber_pb_sb_bad_flag,
 };
 
+static int do_clobber_pb_qconf_wrong_version(char *data)
+{
+	global_super->qconf.version &= 0;
```

Comment:
`&= 0` is the same as `= 0`. Don't make things needlessly complicated here. Just assign `0` to the field directly.